modules/installation-proxy-configure-object.adoc

```diff
@@ -0,0 +1,121 @@
+// Module included in the following assemblies:
+//
+// * installing/insall_config/enable-cluster-wide-proxy.adoc
+
+[id="installation-proxy-configure-object_{context}"]
+= Enabling cluster-wide proxy
+
+The proxy object is used to manage cluster-wide egress proxy. When a cluster is
+installed or upgraded without proxy configured, a proxy object is still
+generated but it will have a nil spec and status. For example:
+
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1
+kind: Proxy
+metadata:
+  creationTimestamp: "2019-08-21T22:36:49Z"
+  generation: 2
+  name: cluster
+  resourceVersion: "24913"
+  selfLink: /apis/config.openshift.io/v1/proxies/cluster
+  uid: 2a344b01-c464-11e9-b3f3-025de4b59c38
+spec:
+  trustedCA:
+    name: ""
+status:
+----
+
+A cluster administrator can configure proxy for {product-title} by modifying
+this `cluster` proxy object.
+
+NOTE: Only the proxy object named `cluster` is supported, and no additional
+proxies can be created.
+
+.Prerequisites
+
+* Cluster administrator permissions
+* {product-title} `oc` CLI tool installed
+
+.Procedure
+
+. Use the `oc edit` command to modify the proxy:
++
+----
+$ oc edit proxy/cluster
```

Comment:
I think the `oc edit` step can come after the ConfigMap step; If the user doesn't need additional CA certificates, that step can still be skipped. Otherwise, the user needs to open another terminal, because for the ConfigMap the original terminal now has an editor open.

Reply:
Oops yep, that should be below, nice catch!